amm/src/main/scala/ammonite/util/Util.scala

```diff
@@ -58,6 +58,19 @@ object Util{
   //irrespective of the OS on which script was written
   def normalizeNewlines(s: String) = s.replace("\r", "").replace("\n", newLine)
 
+  def sanitizePath(s: String) =
```

Comment:
Can we implement this using `NameTransformer.encode` or `decode`? We use those in a few places like https://github.com/lihaoyi/Ammonite/blob/9e249f6d5fd97c56121f11ca77b59071e4a72e25/amm/src/main/scala/ammonite/util/Model.scala#L117, and it's the way the Scala compiler currently does it, so if possible it would nice to be consistent with them